applications/compare_cumulative_psf.py

```diff
@@ -201,12 +201,14 @@ def loadData(datafile):
 
     # Plotting image
     dataToPlot = im.getImageData()
-    visualize.plotHist2D(dataToPlot, bins=80)
+    print(dataToPlot)
```

Comment:
This debug line is probably not necessary

Reply:
Right. Just removed it